BaseDados/Código/DB_English.txt

```diff
@@ -96,18 +96,18 @@ CREATE TABLE realstate (
 	energy_class CHAR,
 	fk_client_id INT,
 	fk_agent_id INT,
-	fk_realstate_type_id INT,
+	fk_realestate_type_id INT,
 	fk_city_id INT,
 	fk_typology_id INT
 );
 
-ALTER TABLE realstate_has_amenities 
-	ADD CONSTRAINT fk_realstate_id FOREIGN KEY (fk_realstate_id) REFERENCES realstate(realstate_id),
+ALTER TABLE realestate_has_amenities 
+	ADD CONSTRAINT fk_realestate_id FOREIGN KEY (fk_realestate_id) REFERENCES realestate(realestate_id),
 	ADD CONSTRAINT fk_amenities_id FOREIGN KEY (fk_amenities_id) REFERENCES amenities(id);
 
 ALTER TABLE imovel
```

Comment:
You forgot to change the name of the table on the ALTER. `imovel` fix to `ALTER TABLE realestate`.